src/specs/FrameManager.spec.ts

```diff
@@ -13,7 +13,10 @@ describe('FrameManager', () => {
         .createSpy('docCreateElement')
         .and.callFake((element: string) => {
           return mocks.frame;
-        })
+        }),
+      head: {
+        appendChild: jasmine.createSpy('headAppend')
+      }
```

Comment:
This is a test fix related to a previous commit that changed how we handle styling the frame-router internals.